module.json

```diff
@@ -27,7 +27,9 @@
   },
   "targetDependencies": {
     "/mbed-os/net/stacks/lwip": {
-      "sal-stack-lwip": "^1.0.0"
+      "sal-stack-lwip": "^1.0.0",
+      "unity": "^2.0.1",
+      "greentea-client": "^0.1.4"
```

Comment:
"^0.1.4" means "==0.1.4" for major version 0, see the "dependencies" section in http://yottadocs.mbed.com/reference/module.html. Probably not the intention here.